screens/Scan/ScanScreen.tsx

```diff
@@ -37,13 +37,26 @@ export const ScanScreen: React.FC = () => {
   function bluetoothIsOffText() {
     return (
       <Text align="center" color={Theme.Colors.errorMessage} margin="0 10">
-        {t(
-          Platform.OS === 'ios' ? 'BluetoothStateIos' : 'BluetoothStateAndroid'
-        )}
+        {t('bluetoothStateMessage')}
```

Comment:
Why were these two clubbed?

Reply:
As part of 613, got the bluetooth state message from rounak, also confirmed by @ravikp  on 20th Standup